chain-tx-enclave-next/mls/Cargo.toml

```diff
@@ -11,11 +11,12 @@ ring = "0.16.15"
 thiserror = "1.0"
 rustls = "0.18"
 x509-parser = "0.8.0-beta4"
+nom = "5.1"
```

Comment:
why is nom needed?

Reply:
need the `nom::Err` error type to embed into the error enum.

Reply:
oh, `InvalidCertificate`